network/load-balancer/concepts.mdx

```diff
@@ -174,6 +186,31 @@ See [balancing-methods](#balancing-methods).
 
 <Concept>
 
+## Private Load Balancer
+
+A Load Balancer is defined as private when you choose the "private" [accessibility](#accessibility) option during Load Balancer creation. A Private Load Balancer has the following characteristics:
+
+- It has no public IP address for sending requests or initiating TCP connections.
+- It only listens to requests or connections sent to its interface(s) on the [Private Network(s)](/network/vpc/concepts/#private-networks) it is attached to. It is not accessible over the public internet.
+- Like a public Load Balancer, it can be attached to up to eight different Private Networks.
+- It can be configured or deleted using the Scaleway API, console, CLI, Terraform or other devtools.
+- It provides its metrics to [Scaleway Cockpit](/observability/cockpit/concepts/#cockpit), even though there is no traffic.
+- It does not allow the use of a Let's Encrypt [certificate](#certificate) - only imported certificates are supported.
+- It does not support multicloud IP addresses for its backend servers, since it is not directly connected to the Internet so routes to them are not guaranteed.
```

Comment:
```suggestion
- It does not support multicloud IP addresses for its backend servers, since it is not directly connected to the Internet. Routes to them are thus, not guaranteed.
```